My_GOT_app/app/src/main/java/com/example/my_got_app/LannisterActivity.java

```diff
@@ -3,11 +3,20 @@
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.recyclerview.widget.GridLayoutManager;
 import androidx.recyclerview.widget.RecyclerView;
-
 import android.os.Bundle;
-
 import java.util.ArrayList;
 
+/**
+ * The LannisterActivity contains a two-columned list of House Lannister
+ * members, each member has is name and an image
+ *
+ * The list of names is saved in a private variable called lann_names
+ * The list of image ids is saved in a private variable called lann_image_ids
+ *
+ * Uses the Grid Layout from RecyclerView
+ */
+
+
 public class LannisterActivity extends AppCompatActivity {
 
     private final String[] lann_names = {"Tywin",
```

Comment:
This is a bit strange, splitting the characters names and the characters images in to 2 separate arrays?
What about creating a class that will hold the information for every "Character"
I might want to someday expand the "Character" class and add some more information - in your solutions you'll create another array for that information?

Reply:
I can see that you have a "Person" class and you're generating the these 2 arrays (lann_names and lann_image_ids so why don't you just initialize an array of "Person" in here?